RadialMenuControl/UserControl/RadialMenu.xaml.cs

```diff
@@ -187,16 +195,87 @@ private void _clearPie(bool storePrevious)
         /// <param name="storePrevious">Should we store the previous pie (for back navigation)?</param>
         public void ChangeToCustomMenu(object s, MenuBase newSubMenu, bool storePrevious)
         {
-            _clearPie(storePrevious);
-            // Redraw
-            Pie.Draw();
-            Pie.UpdateLayout();
-            // TODO use just an auxilary canvas and add custom controls to that
-            newSubMenu.Diameter = Diameter;
-            CustomRadialControlRoot.Children.Add(newSubMenu);
-            newSubMenu.UpdateLayout();
+           
+            if(newSubMenu is ListSubMenu)
+            {
+                CenterButton.Visibility = Visibility.Collapsed;
+                TogglePie();
+
+                listSubMenu = (ListSubMenu)newSubMenu;
+                IList<RadialMenuButton> ListItems = listSubMenu.ListItems;
+
+                Style menuStyle = new Windows.UI.Xaml.Style { TargetType = typeof(MenuFlyoutPresenter) };
+                menuStyle.Setters.Add(new Setter(WidthProperty, this.Width));
+                menuStyle.Setters.Add(new Setter(MaxHeightProperty, listSubMenu.ListHeight));
+                menuStyle.Setters.Add(new Setter(MarginProperty, listSubMenu.ListMargin));
+                MenuFlyout flyoutMenu = new MenuFlyout();
+                flyoutMenu.MenuFlyoutPresenterStyle = menuStyle;
+
+                ToggleMenuFlyoutItem toggle = new ToggleMenuFlyoutItem();
+                toggle.Text = "◀ Back";
+                toggle.Click += ToggleMenu_Click;
+                flyoutMenu.Items.Add(toggle);
+
+                flyoutMenu.Items.Add(new MenuFlyoutSeparator());
+                foreach (RadialMenuButton btn in ListItems)

```

Comment:
This seems super unnatural. We are taking `RadialMenuButton`, buttons designed for the circular pie menu, and using the **information** from them to create `MenuFlyoutItems`. I think you shouldn't define what MenuFlyoutItems go here and instead, allow the user (Mainpage.xaml.cs) define which MenuFlyoutItems it wants.

You can set some style defaults, but we need to leave most styling details to the consuming application.